pkgs/development/cuda-modules/packages/removeStubsFromRunpathHook/removeStubsFromRunpathHook.bash

```diff
@@ -0,0 +1,80 @@
+# shellcheck shell=bash
+
+# Only run the hook from depsBuildBuild when strictDeps is set
+if [[ -n ${removeStubsFromRunpathHookOnce-} ]]; then
+  nixDebugLog "skipping sourcing removeStubsFromRunpathHook.bash (hostOffset=${hostOffset:-0}) (targetOffset=${targetOffset:-0})" \
```

Comment:
`:-0` isn't that potentially a lie?

Reply:
The current logic requires both `hostOffset` and `targetOffset` to be -1 when `strictDeps` is set, which corresponds to `depsBuildBuild`.